src/App.js

```diff
@@ -12,10 +12,35 @@ class App extends Component {
       movies: [],
       singleMovieView: false,
       isLoading: true,
-      singleMovie: []
+      singleMovie: [],
+      filteredMovies: [],
```

Comment:
This is nice touch putting this extra property in to separate all the movies form the ones that fit the search criteria.